src/opm/parser/eclipse/EclipseState/Grid/MULTREGTScanner.cpp

```diff
@@ -97,13 +97,19 @@ namespace Opm {
       Then it will go through the different regions and looking for
       interface with the wanted region values.
     */
-    MULTREGTScanner::MULTREGTScanner(const Eclipse3DProperties& e3DProps,
+    MULTREGTScanner::MULTREGTScanner(const FieldPropsManager& fp_arg,
+                                     const Eclipse3DProperties& e3DProps,
                                      const std::vector< const DeckKeyword* >& keywords) :
-                m_e3DProps(e3DProps) {
+        fp(fp_arg),
+        m_e3DProps(e3DProps) {
 
+#ifdef ENABLE_3DPROPS_TESTING
         for (size_t idx = 0; idx < keywords.size(); idx++)
             this->addKeyword(e3DProps, *keywords[idx] , e3DProps.getDefaultRegionKeyword());
-
+#else
+        for (size_t idx = 0; idx < keywords.size(); idx++)
+            this->addKeyword(e3DProps, *keywords[idx] , e3DProps.getDefaultRegionKeyword());
+#endif
```

Comment:
Is there supposed to be a difference between these two cases (`ENABLE` and `!ENABLE`)?  I don't see a difference here.

Reply:
Yes - they should be different; but this is still *very much* work in progress!